src/makeRobot.js

```diff
@@ -38,7 +38,67 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name,
+    wheels,
+    version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    goForward(step = 1) {
+      if (step < 0) {
+        return this;
+      }
+      this.coords.y += step;
+
+      return this;
```

Comment:
Why do you need to repeat `return this` 2 times? Fix it everywhere 
```suggestion
      if (step >= 0) {
        this.coords.y += step;
      }

      return this;
```